libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -138,6 +138,109 @@ const AP_Param::GroupInfo AP_Landing::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("TYPE",    14, AP_Landing, type, TYPE_STANDARD_GLIDE_SLOPE),
 
+    // @Param: DS_V_FWD
```

Comment:
I think it would be nice to have a LAND_DS_ENABLE parameter, which enables the deepstall parameters. Note that this doesn't necessarily mean a separate object for landing deepstall (although that would be OK too), just separating the parameter tree. I think otherwise we'll end up running out of the 64 available parameters in the var_info at some point